tests/unit/roommanager.spec.js

```diff
@@ -0,0 +1,129 @@
+const fs = require('fs');
+const path = require('path');
+const roommanager = require("../../roommanager")({ on() {} }, require("../../storage"));
```

Comment:
What does this do?

Reply:
Makes it so that we can import the room manager and run it's functions. It's part of the reason of why I'm working on rewriting the room manager now.